src/containers/edit-profile/profile-tab/ProfileTab.tsx

```diff
@@ -62,7 +59,15 @@ const ProfileTab: FC = () => {
   })
 
   const debouncedUpdateProfileData = useDebounce(() => {
-    void dispatch(updateProfileData(data))
+    const payload = {
+      ...data,
+      videoLink: {
+        student: userRole === UserRoleEnum.Student ? data.videoLink : null,
+        tutor: userRole === UserRoleEnum.Tutor ? data.videoLink : null
+      }
+    }
+
+    void dispatch(updateProfileData(payload))

```

Comment:
Consider removing the `void` operator for better maintainability.
[Link](https://sonarcloud.io/project/issues?id=ita-social-projects_SpaceToStudy-Client&pullRequest=2849&issueStatuses=OPEN,CONFIRMED&sinceLeakPeriod=true)

Reply:
Resolved ✅